client/src/components/geneExpression/index.js

```diff
@@ -73,6 +75,26 @@ class GeneExpression extends React.Component {
     };
   }
 
+  _genesToUpper = col => {
+    // Has to be a Map to preserve index
+    let listGenes = col;
+
+    // Extract the array from column if applicable
+    if (typeof col.asArray === "function") {
```

Comment:
recommend deleting this polymorphic arg support, and just make this function _require_ an array as an argument.  Much simpler conceptually.